sdks/go/pkg/beam/testing/passert/equals_test.go

```diff
@@ -35,6 +35,16 @@ func TestEquals_Good(t *testing.T) {
 	}
 }
 
+func TestEquals_Empty(t *testing.T) {
+	p, s := beam.NewPipelineWithRoot()
+	gotC := beam.CreateList(s, []int{})
+
+	Equals(s, gotC)
+	if err := ptest.Run(p); err != nil {
+		t.Errorf("Pipeline failed: %v", err)
+	}
+}
+
```

Comment:
Could we add a test or two for windowing equality? Ideally at least 1 success and 1+ failures